### PR TITLE
Add clone and resize volume cases for qcow2 encryption volume

### PR DESCRIPTION
**Pull request by meinaLi**
Add clone and resize volume cases for qcow2 encryption volume

Signed-off-by: Meina Li <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type q35 virsh.vol_resize.positive_test.luks_encrypt.qcow2_format.non_acl.sparse_capacity virsh.vol_resize.positive_test.luks_encrypt.qcow2_format.non_acl.delta_capacity virsh.vol_clone_wipe.positive_test.luks_encrypt.non_acl.vol_format.qcow2_f
JOB ID     : d3271494d468dedb8eb724fabb48973b1783a1b9
JOB LOG    : /root/avocado/job-results/job-2021-01-20T00.41-d327149/job.log
 (01/12) type_specific.io-github-autotest-libvirt.virsh.vol_resize.positive_test.luks_encrypt.qcow2_format.non_acl.sparse_capacity.dir_pool: PASS (13.90 s)
 (02/12) type_specific.io-github-autotest-libvirt.virsh.vol_resize.positive_test.luks_encrypt.qcow2_format.non_acl.sparse_capacity.fs_pool: PASS (42.20 s)
…
 (10/12) type_specific.io-github-autotest-libvirt.virsh.vol_clone_wipe.positive_test.luks_encrypt.non_acl.vol_format.qcow2_f.no_prealloc.pool_type.dir: PASS (23.09 s)
 (11/12) type_specific.io-github-autotest-libvirt.virsh.vol_clone_wipe.positive_test.luks_encrypt.non_acl.vol_format.qcow2_f.no_prealloc.pool_type.fs: PASS (52.38 s)
(12/12) type_specific.io-github-autotest-libvirt.virsh.vol_clone_wipe.positive_test.luks_encrypt.non_acl.vol_format.qcow2_f.no_prealloc.pool_type.netfs: PASS (25.75 s)
RESULTS    : PASS 12 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 367.32 s
```

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_resize.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def set_secret_value(password, encryption_uuid):
     libvirt.check_exit_status(ret)
 
 
-def create_luks_vol(vol_name, sec_uuid, params):
+def create_luks_vol(vol_name, sec_uuid, params, test):
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_resize.py:142]: update doc string since one additional parameter is added

Comment (meinaLi) [reply libvirt/tests/src/virsh_cmd/volume/virsh_vol_resize.py:142]: Updated

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**